### PR TITLE
Adds new expedition survival boxes and equipment, overhauls drinking sounds + minor loadout tweaks.

### PR DESCRIPTION
**Pull request by Laniakeaaaa**
### Expedition Survival Gear:

_Credits to DeadOS/Navi for the spritework._
### Expedition Survival Kits:

surv survcan2  

- Expedition Kits contain a multitude of different survival equipment to assist teams on away-missions, while also acting as a storage unit.

### New Equipment:

**Survival Knives:**

survknife 

- Serrated Survival Knives found in survival kits, good for stabbing, cutting, you know, knife things.

**Survival Canteens:**

survcan 

- The classic green metal canteens fillable with water or hellmixes.

### Sound Overhaul:

- Allows for multiple different drinking sounds that aren't the horrific slurping sound.
- list()'d so you can add more gulps, and such in the variable.

### Loadouts:

- The Expedition Kit will be available to the Quartermaster and the Supply Team. Another will be available in the Trajan. Due to the fact they start with a rather sharp knife, it'll be costly to balance it.

**Additional Content to Finish:**

**EGS:** 
egs 

- Electronic Guidance System: EGS - An electronically powered guidance system that assists explorers to locate the center of a planetoid or asteroid, useful for navigation in EVA environments.

- EGS Flags/Set-Up - Flags to allow tracking of the specific main area of the guidance system.

### Images:
survive
survive2
deadossprties

### Things to fix:

- [x] Fix drinking sounds not functioning properly when drinking.
- [ ] Add EGS Flags on certain away missions.
- [x] Add a EGS No Signal Sprite.
- [x] Add an EGS direct sprite.
- [x] Balance Survival Knife.

### CODE REVIEW

**Comment by Laniakeaaaa** [comment]
Going to look at how the nuke pinpointer works in order to figure out the EGS GPS thing.

File: code/modules/urist/items/miscitems.dm [hunk 1]
```diff
@@ -440,3 +440,45 @@
...
+	icon_state = "canteen"
+
+
+
+
+
+
+
+
+// /obj/item/weapon/reagent_containers/food/drinks/cans/waterbottle
```
Comment (Glloyd) [inline code/modules/urist/items/miscitems.dm]: Is this bit of commented out code intentionally here?

**Comment by Glloyd** [comment]
If that's intentional, it's good to merge.

File: maps/nerva/Loadout/loadout_accessories.dm [hunk 2]
```diff
@@ -44,4 +44,9 @@
 /datum/gear/tactical/tacticool
 	allowed_roles = null
 
-#undef ARMED_ROLES
\ No newline at end of file
+/datum/gear/survival/survivalkit
```
Comment (Glloyd) [inline maps/nerva/Loadout/loadout_accessories.dm]: this is where your travis error is coming from. Just make it /datum/gear/survivalkit, or place it under an existing category, unless you want to fuck with travis.

File: maps/nerva/Loadout/loadout_accessories.dm [hunk 3]
```diff
@@ -44,4 +44,9 @@
...
+	display_name = "survival kit"
+	path = /obj/item/weapon/storage/box/survivalkit/
```
Comment (Glloyd) [inline maps/nerva/Loadout/loadout_accessories.dm]: remove the / at the end here
```suggestion
	path = /obj/item/weapon/storage/box/survivalkit
```

**Pull request merged**